schemas/schema.json

```diff
@@ -2989,36 +2989,39 @@
                 "type": "object",
                 "description": "Information on a user's available permissions groups",
                 "properties": {
-                    "Groups": {
+                    "groups": {
```

Comment:
We currently return something like:

```json
{
    "Groups": [
        {
            "Name": "demo shipper\n",
            "Code": "TQGBDEMOA"
        }
    ],
    "Count": 1,
    "DateOfRequest": "2023-06-28T10:28:11"
}
```

Reply:
Do we want to rename the returned fields on the backend to reflect these schema changes?